spec/support/star_wars/schema.rb

```diff
@@ -304,7 +304,10 @@ def edge_nodes
 
     connection :newestBasesGroupedByFaction, BaseType.connection_type do
       resolve ->(obj, args, ctx) {
-        Base.order('sum(faction_id) desc').group(:faction_id)
```

Comment:
This query can't be run on psql, @esjee [fixed this](https://github.com/Willianvdv/graphql-ruby/pull/1#issue-238826519)